StreamChatClient_v3/APIClient/Endpoints/Payloads/ChannelListPayload.swift

```diff
@@ -277,7 +274,7 @@ public struct ChannelConfig: Codable {
         self.messageRetention = messageRetention
         self.maxMessageLength = maxMessageLength
         self.commands = commands
-        self.created = created
-        self.updated = updated
+        createdAt = created
+        updatedAt = updated
```

Comment:
We should update the function parameters, too

Reply:
I changed them too, but this I missed.